app/views/fields/belongs_to/_show.html.erb

```diff
@@ -16,5 +16,5 @@ By default, the relationship is rendered as a link to the associated object.
 %>
 
 <% if field.data %>
-  <%= link_to field.data, [Administrate::NAMESPACE, field.data] %>
+  <%= link_to field.associated_resource, [Administrate::NAMESPACE, field.data] %>
```

Comment:
yep - Rails was taking `field.data` and turning it into `field.data.to_s`, in order to get the link text.

I'm not sure `associated_resource` is super clear. I might try changing it to `display_resource` or something.